CODEOWNERS

```diff
@@ -1,15 +1,4 @@
 # CODEOWNERS
 # https://help.github.com/articles/about-code-owners/
 
-# Require Design Systems Team approval for any new packages or this file
-/CODEOWNERS @cultureamp/frontend-avocados
-/packages/ @cultureamp/frontend-avocados
-/draft-packages/ @ghost
-
-# Exclude modifications to existing packages by assigning it to no one
-/packages/hosted-assets @ghost
-
-# Pipelines are maintained by Foundation team FE Engineers
-/.buildkite/ @cultureamp/frontend-ops-approvers
-/docker-compose.yml @cultureamp/frontend-ops-approvers
-/lerna.json @cultureamp/frontend-ops-approvers
```

Comment:
We probably still want to allow the ops team to be able to approve CI changes for us right?

Reply:
I am not sure this specific Github team even exists, I couldn't find it, so it's either gone or secret/invisible. But I also think we - the Kaizen team - own this repo, so any changes are our accountability & responsibility.

Nonetheless, asking for the Ops team's specific expertise in those areas is advised where needed, but we don't require a formal sign-off.

Reply:
Not so much a formal sign-off, but it would be slightly annoying if I opened a PR related to CI changes and someone from Ops approved it but I can't merge because I need someone else from Kaizen to approve it too 😅 Unless that isn't what this does?